packages/serverless/src/domain/use-case/database.ts

```diff
@@ -118,7 +120,32 @@ export async function getDatabases(
 
         const collections = await listCollections(databaseName, actor);
 
-        const deployStatus = (await getLatestTransaction(databaseName, 'deploy'))?.status ?? null;
+        const latestTransaction = await getLatestTransaction(
+          databaseName,
+          'deploy'
+        );
+
+        let deployStatus = latestTransaction?.status ?? null;
+
+        if (latestTransaction && deployStatus === 'pending') {
+          const zkAppTransaction =
+            await MinaNetwork.getInstance().getZkAppTransactionByTxHash(
+              latestTransaction!.txHash!
+            );
```

Comment:
latestTransaction can be null, try avoid null assertion as much as possible

Reply:
fixed